### DISCUSSION TITLE
Any plans to support external keyboards that have animation built in? (Asus Strix Flare II Animate)?

### DISCUSSION THREAD

**Discussion by anthonytyquin** [seerge/g-helper#2599] [Ideas]
Hi Seerge. 
Awesome work with G Helper! Quick one - I have an Asus Strix Flare II Animate external keyboard (link below) and I wondered if you have any plans to support these models moving forward? 

Here's my keyboard:
https://rog.asus.com/au/keyboards/keyboards/pbt-keycaps/rog-strix-flare-ii-model/ 

Many thanks!! 

Tony

**Comment by seerge** [comment]
@anthonytyquin hello, you can submit requests for new hardware in the [issues section](https://github.com/seerge/g-helper/issues). However I see that there is one [pending already](https://github.com/seerge/g-helper/issues/2428)

Unfortunately I don't have that device and can't add it at the moment.

**Reply by anthonytyquin** [reply]
My apologies I didnt find it so I'll check the link. Thanks for the speedy response!
